Makefile

```diff
@@ -87,13 +92,14 @@ selfcheck: ## check that the Makefile is well-formed
 
 ## Localization targets
 
-extract_translations: ## extract strings to be translated, outputting .mo files
-	rm -rf docs/_build
-	cd platform_plugin_ontask && ../manage.py makemessages -l en -v1 -d django
-	cd platform_plugin_ontask && ../manage.py makemessages -l en -v1 -d djangojs
+extract_translations: ## extract strings to be translated, outputting .po files
+	cd $(PACKAGE) && i18n_tool extract
+	mv $(EXTRACTED_DJANGO) $(EXTRACTED_TEXT)
+	if [ -f "$(EXTRACTED_DJANGOJS)" ]; then cat $(EXTRACTED_DJANGOJS) >> $(EXTRACTED_TEXT); rm $(EXTRACTED_DJANGOJS); fi
```

Comment:
Is there a reason why changing makemessages for i18n_tool?

Reply:
The change was only to replicate the way it is done in [XBlocks](https://github.com/eduNEXT/xblock-completion-grading/blob/4c4877c1ce8a54d542a9e85c025f6e9d9e225ccd/Makefile#L99-L106)

Reply:
do we need it though?